resources/puppetlabs/puppetdb/benchmark/samples/facts/pg2.vm.json

```diff
@@ -4798,7 +4798,7 @@
             "apt"
         ],
         [
-            "libjetty9-java",
+            "libjetty10-java",
```

Comment:
Not sure you wanted to make this change here.

Reply:
Was thinking it's a doc so doesn't matter though not sure about that, reverted it